src/lib.rs

```diff
@@ -198,9 +185,31 @@ pub fn initialize(_argc: *const isize, _argv: *const *const *const u8) -> isize
 /// `"arbitrary-derive"` cargo feature.
 #[macro_export]
 macro_rules! fuzz_target {
-    (|$bytes:ident| $body:expr) => {
+    (init: $init:expr, |$bytes:ident| $body:expr) => {
```

Comment:
Using `$init:expr` matches the existing behavior for `$body` but I'm not sure if that's desirable. It permits all sorts of expressions, including those that return values, which may be misleading. This is code that's executed for side-effects, so `$init:stmt` seems more appropriate. However, non-block statements look strange to me in this position (`fuzz_target!(init: let x = 0, |data| {})` without semicolon would be valid but meaningless). So maybe enforce that it's a block with `$init:block` and expand to `let _: () = $init;` so that it's required to not return anything meaningful?

Reply:
I'll wait and see what the maintainers have to say, but I get your point, still I don't have a preference.